.knip.jsonc

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://unpkg.com/knip@5/schema.json",
+  "ignore": [
+    "**/dist/*",
+    "docs/**/*",
+    // used for `#is-development` conditional import
+    "packages/**/{true,false}.ts",
+    // file acts as a type test
+    "packages/xstate-svelte/test/interpreterAsReadable.svelte"
+  ],
+  "ignoreDependencies": [
+    "@tsconfig/svelte",
+    "@xstate-repo/jest-utils",
+    "@xstate-repo/jest-utils/setup",
+    "babel-preset-solid",
+    "husky",
+    "lint-staged",
+    "svelte-jester",
+    "synckit",
+    "tslib"
```

Comment:
why this one has been ignored? do we explicitly depend on this anywhere? that might be a mistake

Reply:
It's also in `resolutions`:

https://github.com/statelyai/xstate/blob/9841fab65f65d76349386b9489413f8fd134ee84/package.json#L89